core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.kt

```diff
@@ -218,4 +232,57 @@ abstract class CoreMainActivity : BaseActivity(), WebViewProvider {
   }
 
   abstract fun openPage(pageUrl: String, zimFilePath: String = "")
+
+  private fun checkForRateDialog() {
```

Comment:
ah of course, this feature is just a chunk of random code.

Can we get this in a class? `RateUsFeature`?

Reply:
haha! good idea.

Reply:
It will need a DialogShower so it will need to be Activity scoped  I would think